JungleBus/Aws/Sqs/SqsQueue.cs

```diff
@@ -87,7 +85,7 @@ public SqsQueue(RegionEndpoint endpoint, string queueName, int retryCount, IMess
                 _simpleQueueService.SetQueueAttributes(createResponse.QueueUrl, new Dictionary<string, string>() { { "MessageRetentionPeriod", "1209600" } });
             }
 
-            MessageParser = messageParser;
+            MessageParser = new MessageParser();
             MaxNumberOfMessages = 10;
```

Comment:
Make a parameter with a default value? Don't hard code this, push this out to config?